### PR TITLE
Add a whitelist entry for Greg J

### PR DESCRIPTION
**Pull request by gjuggler**
Hi folks,

Sean pointed me here to gain access to the Terra app. I'm a new Terra / All of Us team member at Verily and I'd like to kick the tires a bit.

Side note: it seems a bit laborious to be adding testers one-by-one to a config file within the UI codebase. Has there been a thought to control beta access via a Google Group, and/or enforce on the backend instead of the UI?

### CODE REVIEW

**Comment by gjuggler** [comment]
I realized I might require access via my google.com account as well (to view Baseline data), so I added another MD5 to the list.

**Review by panentheos — APPROVED** [review]
Thanks Greg. The whitelist is a temporary measure, so we went with something quick and easy, but we are evaluating other methods for access control.

**Comment by gjuggler** [comment]
Thanks @panentheos. Could you merge this when you have a chance? I fixed merge conflicts, but I don't have write access to the repo.

**Pull request merged**
